.jshintignore

```diff
@@ -0,0 +1,6 @@
+src/libs/potrace.js
```

Comment:
We really should use the [publicly maintained version of this lib for node](https://www.npmjs.com/package/potrace) instead of packaging one with the app. The only reason I didn't do this for most of these is because there isn't really a good (or appropriate) node/npm lib available for them.